lib/ex_spirit/parser/text.ex

```diff
@@ -530,13 +531,13 @@ defmodule ExSpirit.Parser.Text do
                 }
               parser when is_function(parser, 1) ->
                 %{context |
-                  position: context.position + byte_size(<<c::utf8>>),
+                  position: context.position,
                   column: if(c===?\n, do: 1, else: context.column+1),
                   line: context.line + if(c===?\n, do: 1, else: 0),
```

Comment:
Also wtf was I thinking here, can you delete `column` and `line` lines here too, these cases are only for when `c` is used, and as we are in the initial `nil` branch, then it is obviously not used, so these should be no-ops too!  o.O